lms/static/scripts/frontend_apps/config.ts

```diff
@@ -225,7 +226,7 @@ export type ConfigObject = {
     speedGrader?: SpeedGraderConfig;
   };
   contentBanner?: ContentBannerConfig;
-  grading?: GradingConfig; // Only present for instructors
+  instructorToolbar?: InstructorConfig;
```

Comment:
I feel `instructorToolbar` couples a bit the config with the place in which we plan to use it now, but it might be something we want to use somewhere else. What if we call it `instructorConfig` instead?

Reply:
> I feel instructorToolbar couples a bit the config with the place in which we plan to use it now, but it might be something we want to use somewhere else.

Currently the backend knows whether the instructor toolbar should be enabled or not and only sets the config if it is. If we want to reuse parts of this config in other contexts (eg. the course and assignment name), we'll need to do a larger refactoring in any case.

Reply:
Fair enough